src/firestore/firestore.ts

```diff
@@ -110,7 +110,16 @@ export function getFirestoreOptions(app: FirebaseApp): any {
 
 function initFirestore(app: FirebaseApp): Firestore {
   const options = getFirestoreOptions(app);
-  // Lazy-load the Firestore implementation here, which in turns loads gRPC.
-  const firestoreDatabase: typeof Firestore = require('@google-cloud/firestore');
+  let firestoreDatabase: typeof Firestore;
+  try {
+    // Lazy-load the Firestore implementation here, which in turns loads gRPC.
```

Comment:
Do we need to do the same error checking for Storage?

Reply:
We already have it in Storage: https://github.com/firebase/firebase-admin-node/blob/c6a4e222d8c60eeb03e22e997e0cc36d9f25d28d/src/storage/storage.ts#L61-L71

Reply:
I made an improvement by including the original error in the returned error message.